mne/viz/_brain/_timeviewer.py

```diff
@@ -418,8 +418,33 @@ def __init__(self, brain):
         self.set_slider_style(playback_speed_slider)
         self.set_slider_style(time_slider)
 
+        # setup key bindings
+        self.key_bindings = {
+            '?': self.help,
+            'y': self.toggle_interface,
+            't': self.apply_auto_scaling,
+            'u': self.restore_user_scaling,
+            ' ': self.toggle_playback,
+        }
+
+    def keyPressEvent(self, event):
+        from PyQt5 import QtCore
+        if event.key() == QtCore.Qt.Key_questiondown:
```

Comment:
This works for me on both macOS and Linux

Reply:
(BTW if I do `print(event.key())` and `print(type(event.key())` on macOS I get `63` and `int`, so it's possible that using the ASCII char code would have also be enough :) )

Reply:
But what's confusing to me is that `QtCore.Qt.Key_questiondown == 191`

Reply:
Oh actually it's not being routed through this part of the conditional at all, it's going through the `callback=self.key_mindings.get(event.text())`

Reply:
@hoechenberger tried both `event.text()` and `Key_Question` without success.